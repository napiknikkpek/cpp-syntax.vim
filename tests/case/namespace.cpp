foo::bar::a;
namespace {}
namespace foo {}
namespace foo::bar {}
