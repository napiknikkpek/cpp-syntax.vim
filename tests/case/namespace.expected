ttt..ttt..x.
aaaaaaaaa...
aaaaaaaaa.ttt...
aaaaaaaaa.ttt..ttt...
